CHANGELOG.md

```diff
@@ -42,14 +42,23 @@ straightforward as possible.
 
 -
 
-## [v1.1.3] - 2023-11-29
+## [v1.1.4] - 2023-12-07
 
-Here we write upgrading notes for brands. It's a team effort to make them as
-straightforward as possible.
+You can install this version by running `pip install rocketpy==1.1.4`
+
+### Fixed
+
+- DOC: Change from % to ! in the first cell to run properly in Colab. [#496](https://github.com/RocketPy-Team/RocketPy/pull/496)
```

Comment:
This line is not necessary for changelog. I'm going to update the changelog description to avoid future mistakes.

PLease don't accept suggestion

```suggestion
```